build.gradle

```diff
@@ -102,8 +101,7 @@ ext.libraries = [
         commons_cli          : "commons-cli:commons-cli:${versions.commons_cli}",
         thrift               : "org.apache.thrift:libthrift:${versions.thrift}",
         kafka_clients        : "org.apache.kafka:kafka-clients:${versions.kafka}",
-        kafka                : "org.apache.kafka:kafka_2.12:${versions.kafka}",
-        zkclient             : "com.101tec:zkclient:${versions.zkclient}"
+        kafka                : "org.apache.kafka:kafka_2.12:${versions.kafka}"
```

Comment:
Curious, which class is getting used from this library?

Reply:
Hi @adkharat 

I just checked I see below classes that you mentioned are coming from `kafka-clients-3.9.0.jar` nothing from these are from `org.apache.kafka:kafka_2.12:${versions.kafka}`

```
org/apache/kafka/clients/producer/KafkaProducer.class
org/apache/kafka/clients/admin/AdminClient.class
org/apache/kafka/common/errors/TopicExistsException.class
org/apache/kafka/common/serialization/ByteArraySerializer.class
```
So I am curious do we even need `"org.apache.kafka:kafka_2.12:${versions.kafka}"` in this upgrade?